bsb/placement/arrays.py

```diff
@@ -94,7 +94,7 @@ def place(self, chunk, indicators):
                 for c in unique_chunks_list:
                     idx = np.where((chunks_list == c).all(axis=1))
                     pos_current_chunk = positions[idx]
-                    self.place_cells(indicator, pos_current_chunk, chunk=c)
+                    self.place_cells(indicator, [pos_current_chunk], chunk=c)
```

Comment:
I think it was correct before:

```suggestion
                    self.place_cells(indicator, pos_current_chunk, chunk=c)
```